lib/ash/policy/authorizer/authorizer.ex

```diff
@@ -356,6 +356,20 @@ defmodule Ash.Policy.Authorizer do
     entities: [
       @field_policy_bypass,
       @field_policy
+    ],
+    schema: [
+      private_fields: [
+        type: {:one_of, [:show, :hide, :include]},
+        default: Application.compile_env(:ash, :policies)[:private_fields] || :show,
+        doc: """
+        Option for how private fields should be handeled by field policies in internal functions.
+
+        - :show will always show private fields
+        - :hide will always hide private fields
+        - :include will let you to write field policies for private fields and private fields
```

Comment:
We will want to have this as a single line, and tell users to consult the policies guide for more. Then, put this information there. Large option documentation blocks doesn't look good in the generated docs.

Other than that, this LGTM 🥳